_posts/2020-10-01-october-meetup.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "October 2020"
+date:   2020-10-01 00:00:00
```

Comment:
The date here and the date in the filename have to be in the past for Jekyll to pick it up. Sorry, it's confusing, it makes sense to put the date of the actual CocoaHeads, but that's not how the blog engine of GitHub Pages works :/

Reply:
Gotcha. Thanks for the quick review. Fixing!